src/mscorlib/src/System/Reflection/Assembly.cs

```diff
@@ -536,8 +556,13 @@ public static Assembly LoadFile(String path)
 
             AppDomain.CheckLoadFileSupported();
 
+#if FEATURE_CORECLR
+            AssemblyLoadContext alc = new FileLoadAssemblyLoadContext();
```

Comment:
Same change for `Load(byte[])`? `Load(byte[])` was public for historic reasons, but it was not in reference assemblies and tested - it should be consistent with LoadFile.